### PR TITLE
Pack containers into GitHub CI

### PR DESCRIPTION
**Pull request by mangelajo**
These .tar.gzs later will be used by common non-podman capable builders
(copr) to create rpms with the container images.

Signed-off-by: Miguel Angel Ajo <majopela@redhat.com>

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -304,6 +304,9 @@ if [ $NIGHTLY -eq 1 ]; then
   exit 0
 fi
 
+# create container tar.gzs for the consumed container images for non-nightly releases
+ARCHITECTURES="amd64 arm64" OUTPUT_DIR="${STAGING_DIR}" sudo -E "${ROOT}/packaging/image/components/archive.sh" || exit 1
```
Comment (mangelajo) [inline scripts/release.sh:308]:
```suggestion
ARCHITECTURES="amd64 arm64" OUTPUT_DIR="${STAGE_DIR}" sudo -E "${ROOT}/packaging/image/components/archive.sh" || exit 1
```

File: Makefile [hunk 2]
```diff
@@ -218,6 +218,14 @@ build-containerized-all-in-one-iptables-arm64:
 	+$(MAKE) _build_containerized_aio ARCH=arm64 IPTABLES=iptables
 .PHONY: build-containerized-all-in-one-iptables-arm64
 
+###############################
+# container image packaging   #
+###############################
+
+image-tars:
```
[the remark this answers is not in the document]
Comment (mangelajo) [reply Makefile]: Done, I used `tar-ocp-containers` since package is something we may want to use for a final rpm.

**Comment by copejon** [comment]
openshift-ci is having some internal difficulties.  will re-run once it's stable again.

**Pull request merged**